public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -1478,7 +1478,7 @@ export const Explorer = ({
         query,
       }}
     >
-      <div className="dscAppContainer">
+      <div className={`dscAppContainer${uiSettingsService.get('theme:darkMode') && ' explorer-dark'}`}>
```

Comment:
i don't remember if `theme:darkMode` is a boolean, if so then when `uiSettingsService.get('theme:darkMode')` is `false`, the className would be `dscAppContainerfalse`?

Reply:
it is boolean true when under dark mode, but is empty when not under dark mode therefore should not be seeing 'dscAppContainerfalse' case.